python/ray/tune/tests/test_api.py

```diff
@@ -973,6 +973,64 @@ def _restore(self, state):
             self.assertEqual(trial.status, Trial.TERMINATED)
             self.assertTrue(trial.has_checkpoint())
 
+    def testLogToFile(self):
+        def train(config, reporter):
+            import sys
+            from ray import logger
+            for i in range(10):
+                reporter(timesteps_total=i)
+            print("PRINT_STDOUT")
+            print("PRINT_STDERR", file=sys.stderr)
+            logger.info("LOG_STDERR")
+
+        register_trainable("f1", train)
+
+        # Do not log to file
+        [trial] = tune.run("f1", log_to_file=False).trials
```

Comment:
another test I'd add is with a Trainable class with run(log_to_file=True, reuse_actors=True).

We will eventually want to support reuse_actors for the function API, and it would be good to validate that the current implementation works out of the box.

Reply:
Done, but noticed some weird behavior. Even when reusing actors, first all actors are initialized separately. After that, only one actor is reused over and over. I'll open an issue for that.